### ISSUE TITLE
Testing feedback: adding a quote

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#1548]
As the tests come in for Gutenberg I am going to make issues but also add feedback here as a constant thread for people to check.

For reference the tests are here: https://make.wordpress.org/test/2017/06/27/call-for-testing-gutenberg/

This feedback is on this task:

"The true sign of intelligence is not knowledge but imagination. - Albert Einstein"

**Comment by karmatosed** [comment]
"I still can't figure out why "enter" isn't working to leave the list from before so I'm deleting the list and making it text, and then made it a quote. I like how easy it is to change formatting, it'd be better still if there were keyboard shortcuts, or if there are if hovering over the icons would reveal what the shortcuts are."

**Comment by karmatosed** [comment]
"It's easy, but need lots of editing. I couldn't, for example, add a proper curly quote from the interface."

**Comment by karmatosed** [comment]
"Is slightly confusing because I selected "pull quote" first before realising there was another option."

**Comment by karmatosed** [comment]
"WHY WON'T YOU LET ME USE THE MOUSE TO DRAG + COPY? I pasted too much into the quote, and trying to get the name into the citation was IMPOSSIBLE without resorting to deleting letter by letter. I kept getting multiple blocks highlighted."

**Comment by jwold** [comment]
> WHY WON'T YOU LET ME USE THE MOUSE TO DRAG + COPY?

Couldn't quite understand what they were saying here. Are they trying to drag selected text up and down?

**Comment by karmatosed** [comment]
"Same problem with the 'Write' block as above."

**Comment by karmatosed** [comment]
> Couldn't quite understand what they were saying here. Are they trying to drag selected text up and down?

My suspect on this is that they are having some drag and drop bug. A good point to note is there are pretty raw but useable tests none the less. Later tests we need to get more information and videos. However, the goal now is exposure.

**Comment by karmatosed** [comment]
"It wasn't until adding the Quote that clicking on the plus sign reveals more options, instead of the text/image options that show up on hover. Most people will only choose the options they see on Hover, not even thinking that clicking the  Plus sign would allow for things like blockquote and lists. The options that show up on clicking the plus sign should be the options that show up on hover."

**Comment by karmatosed** [comment]
"Does the quote design come from theme or editor? It would be great if it could be matched with the theme."

**Comment by karmatosed** [comment]
"Very simple.  It helps that the quotation block already comes up looking like a quote so you know exactly what to put and where."

**Comment by karmatosed** [comment]
"I like how this works. With the citation built in separately. This will definitely allow for better default markup. "

**Comment by karmatosed** [comment]
"same as current WP. The problem is that if you make a mistake you have to erase the whole thing and start again. Currently it is easier. In fact I did not see the citation part until the next question."

**Comment by karmatosed** [comment]
"This one was easier because I saw the "Quotation" in the + list."

**Comment by karmatosed** [comment]
"After the experience of adding lists and headings, I knew how to add a quote, so I found it very easy. It's nice that Gutenberg offers two styles to choose from. The only thing I'd maybe add is the option to outdent the quote, so that it stands out more."

**Comment by karmatosed** [comment]
"The caption "Write citation ..." is in the wrong field. I would move it above where the cursor is blinking. In place of "Write citation ..." I would add "Add author here"."

**Comment by karmatosed** [comment]
"Again, I really struggled to get the "Insert Block" button to display the window with my options in a place where I could see them on my iPad. In addition, the pop-up menu for the list block (with different list options and text formatting) remained on screen, making everything look quite busy.

Once the Quote block was added, adding the quote worked very well and I was tickled to see the "Write citation..." section so I didn't have to mess with HTML to add a citation to the quote. It was a pretty small tap target on my iPad but I managed after a couple taps."
